weave/upload.py

```diff
@@ -256,7 +256,10 @@ def sanitize_upload_basket_kwargs(self):
         # I think it is wise to ignore pylint here because we should only
         # set self.file_system *after* we have sanitized it.
         # pylint: disable-next=attribute-defined-outside-init
-        self.file_system = self.kwargs.get("file_system", get_file_system())
+        self.file_system = self.kwargs.get("file_system", None)
+        if self.file_system is None:
+            # pylint: disable-next=attribute-defined-outside-init
+            self.file_system = get_file_system()
         # pylint: disable-next=attribute-defined-outside-init
```

Comment:
3 WHOLE PYLINT DISABLES?????

Reply:
I remember having this problem when I worked on this stuff as well. I am not sure that there's a better way to do this.